src/components/common/LandingLayout.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { element } from 'prop-types';
+import { ReactComponent as Iphone6 } from 'assets/iphone6.svg';
+import { ReactComponent as AppStoreBtn } from 'assets/appstore.svg';
+import { ReactComponent as FacebookBtn } from 'assets/facebook.svg';
+import { ReactComponent as TwitterBtn } from 'assets/twitter.svg';
+
+const LandingLayout = ({ leftSideElement, rightSideElement }) => {
+  return (
```

Comment:
As a suggestion, you can return explicit here 
```suggestion
const LandingLayout = ({ leftSideElement, rightSideElement }) => (
```